### DISCUSSION TITLE
Change mouse cursor on hover

### DISCUSSION THREAD

**Discussion by Septias** [emilk/egui#1430] [Q&A]
Is it currently possible to change the mouse-cursor to say `pointer` when an element is hovered?

**Comment by emilk** [comment]
You can change the cursor icon with https://docs.rs/egui/latest/egui/struct.PlatformOutput.html#structfield.cursor_icon
If you want to show text where the cursor is, use the `Painter` API and just draw some text.

**Comment by Septias** [comment]
Is there a way to get the context object in a deeply nested ui-function structure? We use some utility-functions to create the ui and only pass `ui` in the arguments, so I have no direct way to use `https://docs.rs/egui/latest/egui/struct.Context.html#method.output`

**Reply by emilk** [reply]
https://docs.rs/egui/latest/egui/struct.Ui.html#method.ctx - I can really recommend reading through the docs and examples. You will learn a lot!

**Reply by Septias** [reply]
Uh, I'm sorry that is quite an obviouse one. I skimmed for that method but only looked for `context()` and not ctx().
